specify/specify7
# Unset auto width for forms when not in subForm to avoid shifts

Fixes #4738

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

- Go to data entry
- Click on taxon
- Input a parent of taxon
- Delete what you selected and reenter a new taxon

- [ ] See there is no form shift
- [ ] test other places with forms, other data entry forms, app resources, record sets, interactions. Verify width is ok
